### PR TITLE
Update oncoprint with focal CN data

### PR DESCRIPTION
**Pull request by cbethell**
### Purpose/implementation

This PR incorporates the cnv data into the data represented on the oncoprint produced in PR #176.

It builds onto the `01-plot-oncoprint.R` and `run-oncoprint.sh` scripts in the oncoprint-landscape module. 

### Issue

This PR addresses issue #6 on producing an oncoprint that displays the landscape of genetic lesions across PBTA.

### Directions for reviewers

- In this PR, I updated `run-oncoprint.sh` to include the `annotated_cn.tsv` file produced in the focal-cn-file-preparation module as mentioned [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/186#issuecomment-549839050). With this addition, the sample ids on the plot become illegible. Should I remove them from the plot in this PR or wait to see what happens as we update the `annotated_cn.tsv` file?

Note: The labels may change as the `annotated_cn.tsv` file is updated to use `gain` and `loss` broadly as noted [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/216/files#r341868719). 

### Results

See the current plot below:
maf_oncoprint

#### Docker and continuous integration
- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### PR Checklist
* [x] Run a linter
* [x] Set the seed- Not needed.
* [x] Comments and/or documentation up to date 
* [x] Double check your paths
* [x] Spell check any Rmd file or md file
* [x] Restart R and run all notebooks fresh and save

### CODE REVIEW

**Comment by cbethell** [comment]
>Looks good to me! I had one very trivial formatting comment. I do think we might eventually want to take out the sample names, but I think that could wait. The other thing that comes to mind, which is outside the scope of this pull request is thinking about what functionality you might add to produce plots for individual disease types.

Sounds good. I will begin thinking about the functionality needed to produce plots for individual disease types and file a PR, more than likely after we have the updated focal CN file.

**Comment by jaclyn-taroni** [comment]
Agreed

**Comment by jharenza** [comment]
Hi @cbethell - looks like this is coming along! One thing I noticed was the number of samples in your oncoplot being 2000. The total N = 943 patients, and I believe maybe we wanted to go lower, based on primary only: #167 and #171. Based on the mutation spread, it looks like the RNA and DNA samples are being treated as separate specimens, so maybe check out how these are being merged such that the RNA samples have the same `Tumor_Sample_Barcode` as their DNA counterparts (this would be patient-level).

**Comment by cbethell** [comment]
> Hi @cbethell - looks like this is coming along! One thing I noticed was the number of samples in your oncoplot being 2000. The total N = 943 patients, and I believe maybe we wanted to go lower, based on primary only: #167 and #171. Based on the mutation spread, it looks like the RNA and DNA samples are being treated as separate specimens, so maybe check out how these are being merged such that the RNA samples have the same `Tumor_Sample_Barcode` as their DNA counterparts (this would be patient-level).

@jharenza Thank you for this comment. I'll revisit how these samples are merged and hopefully get to the bottom of it by the end of the day.

**Comment by jaclyn-taroni** [comment]
That's a very good point @jharenza - @cbethell this has the same issues that you encountered with https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/240 (and I forgot which context we talked about it). So you will need to map to _participant_ identifiers to make these plots, perhaps after filtering to primary or primary/plus as @jharenza suggests.

**Comment by cbethell** [comment]
Here is the most recent plot produced after filtering to primary samples and attempting to assign the same `Tumor_Sample_Barcode` to the `RNA` samples as their `DNA` counterparts:

**Comment by jaclyn-taroni** [comment]
Since I made the most recent changes in this pull request, I'm going to request @cansavvy for a review.

File: analyses/oncoprint-landscape/run-oncoprint.sh [hunk 1]
```diff
@@ -3,19 +3,88 @@
...
+wget -N --directory-prefix=driver-lists -O driver-lists/brain-goi-list-short.txt https://github.com/marislab/create-pptc-pdx-oncoprints/raw/2c9ed2a2331bcef3003d6aa25a130485d76a3535/data/brain-goi-list.txt
+
+#### Download consensus mutation files
+
+# TODO: remove once the consensus files are included in the data download
+# wget only the newer files
+wget -N https://open-pbta.s3.amazonaws.com/data/snv-consensus/snv-consensus_11122019.zip
+unzip -u snv-consensus_11122019.zip
+# remove the mac specific directory included in the zip file
+rm -r __MACOSX
```
[the remark this answers is not in the document]
Comment (jaclyn-taroni) [reply analyses/oncoprint-landscape/run-oncoprint.sh]: No, this gets run within the Debian-based Linux container without issue. The inclusion of this directory stems from how you compressed the files before putting them on S3.

[the remark this answers is not in the document]
Comment (cansavvy) [reply analyses/oncoprint-landscape/run-oncoprint.sh]: Cool.

**Comment by jharenza** [comment]
> Here is the most recent plot produced after filtering to primary samples and attempting to assign the same `Tumor_Sample_Barcode` to the `RNA` samples as their `DNA` counterparts:
> 
> 

One comment on this is that it looks like there are more CN gains/losses than we would expect in brain tumors, but this may be because we are setting broad and low thresholds for these. Typically. we want to see only focal amplifications and deep losses here, so after we get the consensus calls, we should revisit what we show in terms of CN.

**Comment by jaclyn-taroni** [comment]
@jharenza that plot is now out of date and the new one looks worse in that regard. I don't expect these plots to be particularly useful yet and have added a note to the README: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/243/files#diff-600b9e50568dec2048054763c2bed9f4R3

**Pull request merged**